src/components/lyne-link/styles/lyne-link.component.scss

```diff
@@ -0,0 +1,70 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin, available for shared
+// and default mode
+@include host-component-properties;
+
+:host {
+  display: inline-block;
+
+  --text-color-normal: var(--color-metal-default);
+  --text-color-hover-focus: var(--color-red125-default);
+  --text-color-active: var(--color-red150-default);
+
+}
+
+.link--negative {
+  --text-color-normal: var(--color-white-default);
+  --text-color-hover-focus: var(--color-cloud-default);
+  --text-color-active: var(--color-silver-default);
+}
+
+.link {
+  display: flex;
+  align-items: center;
+  gap: pxToRem(var(--spacing-fixed-1x));
+  color: var(--text-color-normal);
+  text-decoration: none;
+}
+
+.link--icon-placement-right {
+  flex-direction: row-reverse;
+}
+
+.link:active {
+  color: var(--text-color-active);
+}
+
+.link:focus,
+.link:hover {
+  color: var(--text-color-hover-focus);
+}
+
+.link__text_icon {
+  width: pxToRem(var(--size-icon-ui-small));
+  height: pxToRem(var(--size-icon-ui-small));
+}
+
+.link[dir="rtl"].link--icon-flip .link__text_icon {
+  transform: scaleX(-1);
+}
+
+.link__text_icon slot {
+  fill: currentColor;
+
+  @include ifForcedColors {
```

Comment:
wouldn't it make more sense to include it on the link color itself? the slot's fill will be inhertied then via currentColor, right?